src/modules/dashboard/components/ColonyHome/TabTransactions/TabTransactions.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import { ColonyType } from '~immutable/index';
+
+import Transactions from '~admin/Transactions';
+
+interface Props {
+  colony: ColonyType;
+}
+
+const TabTransactions = ({ colony }: Props) => (
+  <Transactions colonyAddress={colony.colonyAddress} />
```

Comment:
Since this `TabTransactions` component is simply returning the `Transactions` component, it seems a little unnecessary. Maybe we should just remove this `TabTransactions` component and use `Transactions` directly in the `ColonyHome` component. Thoughts?